spec.bs

```diff
@@ -1873,3 +1804,21 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   1. If |type| is "`pattern`" then return |strippedValue|.
   1. Return the result of running [=canonicalize a hash=] given |strippedValue|.
 </div>
+
+<h2 id=acknowledgments class=no-num>Acknowledgments</h2>
+
+The editors would like to thank
+Anne van Kesteren,
+Luca Casonato, and
+Sayan Pal
+for their contributors to this specification.
+
+A special thanks to the [pillarjs/path-to-regexp](https://github.com/pillarjs/path-to-regexp) creators for their work on the open-source library and pattern syntax which inspired this standard.
+
+This standard is written by
+Ben Kelly ([Google](https://www.google.com/), [wanderview@chromium.org](mailto:wanderview@chromium.org)),
+Jeremy Roman ([Google](https://www.google.com/), [jbroman@chromium.org](mailto:jbroman@chromium.org)), and
+Shunya Shishido ([Google](https://www.google.com/), [sisidovski@chromium.org](mailto:sisidovski@chromium.org)).
```

Comment:
These should stay synced with https://github.com/whatwg/sg/pull/218/files ; if either changes let's be sure to update the other PR.